backend/code_review_backend/issues/api.py

```diff
@@ -172,6 +174,18 @@ def get_queryset(self):
             .order_by("-created")
         )
 
+        # Display only publishable issues by default
+        publishable = self.request.query_params.get("publishable", "true").lower()
+        _filter = Q(in_patch=True) | Q(level=LEVEL_ERROR)
```

Comment:
We are duplicating this filter in many places, when we change the publishable check we'll have to remember to update all instances

Reply:
Oh believe me, i'm well aware :cry: